src/main/service/GeocodingService.ts

```diff
@@ -1,14 +1,20 @@
-import { Entry, Geocoder } from 'node-geocoder';
+import { Entry, Geocoder, Query } from 'node-geocoder';
 import { GeoCoordinate } from '../data/GeoCoordinate';
 
+export interface GeocoderConfig {
+  geocoder: Geocoder;
+  countryCode?: string;
+}
```

Comment:
Maybe a comment here why we need country code? and in what scenario its undefined